docs/contents.md

```diff
@@ -47,8 +47,8 @@ When creating notification rules, selecting **Manual Input** as a notification t
 
 | Playbook Name          | Description                                                                                                                                   |
 |:-----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|
-| Create Indicator       | Extracts indicators from the text provided as input                                                                                           |
-| Create Alert           | Creates an alert based on the values provided in the form                                                                                     |
+| Create Indicator       | Extracts indicators from the text provided as input. Gets triggered on createIndicator command when used in Microsoft Teams                   |
+| Create Alert           | Creates an alert based on the values provided in the form. Gets triggered on createAlert command when used in Microsoft Teams                 |
```

Comment:
Creates an alert based on the values provided in the form and is triggered when `createAlert` command is sent through Microsoft Teams.